### PR TITLE
Fixes cargo admin manager

### PR DESCRIPTION
**Pull request by MaxIsJoe**
Title

For the future, if you want to add commands for admins make it inside the AdminCommandsManager which is similar to the PlayerCommandsManager

### CODE REVIEW

File: UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs [hunk 1]
```diff
@@ -490,5 +493,48 @@ public static void LogAdminAction(string msg, string userName = "")
 		}
 
 		#endregion
+
+		#region CargoControlCommands
+
+		[Command(requiresAuthority = false)]
+		public void CmdRemoveBounty(int index, bool completeBounty)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs]: security concerns?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs]: fixed now?

File: UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminBountyManager.cs [hunk 2]
```diff
@@ -43,7 +52,7 @@ private void OnDisable()
 		public override void OnEnable()
 		{
 			base.OnEnable();
-			RefreshBountiesList();
+			AdminCommandsManager.Instance.CmdRequestCargoServerData(PlayerManager.PlayerScript.connectedPlayer.UserId);
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminBountyManager.cs]: I don't think you need to specify it I think it Fills it out automatically, not to sure the need to ask noonly

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminBountyManager.cs]: It works so sure, but if the entries dont get updated after we merge still then we know where the issue is.

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**